src/utils/common.ts

```diff
@@ -62,6 +62,7 @@ export function createTurboSigner({
     case 'ethereum':
     case 'pol':
     case 'matic':
+    case 'base-eth':
```

Comment:
nit: remove default in switch to have compiler catch changes before runtime (faster updates)